src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -267,7 +269,7 @@ private void handleKeys(boolean isForwardKey) {
     }
 
     private void scrollandShowPanel(int selectedPanelIndex, int numOfPanels) {
-        ui.getMenuControl().scrollTo(selectedPanelIndex, numOfPanels);
+        ui.scrollTo(selectedPanelIndex, numOfPanels);
```

Comment:
As I moved it to `PanelControl`, I combine `scrollTo` with `scrollAndShowPanel` as `scrollToPanel` and I have changed all the parameters so that now only 1 parameter, `panelIndex` since we always use number of panels as the second parameter.